db/new_seeds/scenarios/participants/training_record_states.rb

```diff
@@ -38,6 +39,8 @@ def fip_school_with_previous_fip_cohort
                                                      .new(name: "FIP School with previous FIP induction programme cohort")
                                                      .build
                                                      .with_an_induction_tutor(full_name: "FIP School with previous FIP induction programme cohort SIT", email: "fip-previous-fip-programme-cohort@example.com")
+                                                     .with_partnership_in(cohort: previous_cohort)
+                                                     .with_partnership_in(cohort:)
```

Comment:
not sure I understand why we're partnering in both previous and current for previous cohort fip?

Reply:
Hi @cwrw there's a slight difference but it also creates partnerships..